### DISCUSSION TITLE
Can you release the 2.8.0 version of kong with latest commit in master branch ?

### DISCUSSION THREAD

**Discussion by TACY-octo** [Kong/kong#9068] [Help] [answered]
Hello ! 

I have multiple error in kong have been fixed in master branch (https://github.com/Kong/kong/issues/8762), do you think you can create a release with the last change in master branch for the version 2.8.0 ?

And create a update for the kong chart ? 

Best regards,

**Comment by hanshuebner** [comment]
Hello @TACY-octo,

thank you for your question.  The [fix you've referred to](https://github.com/Kong/kong/pull/8690) will be included in the upcoming release 3.0 of Kong gateway.  At this point, we don't have plans to backport it to the 2.8 line, sadly.

Kind. regards,
Hans

**Reply by TACY-octo** [reply]
Hello @hanshuebner,

When do you expect to release version 3.0.0?
And why the old 2.4.x has fixed were released and not the 2.8.x version (it's just to understand how Kong is delivered :))

Best regard,

Cyril

**Comment by hanshuebner** [comment, marked as answer]
> When do you expect to release version 3.0.0?

3.0.0 is scheduled to be released in Q3/2022, but I cannot provide you with an exact release date.

> And why the old 2.4.x has fixed were released and not the 2.8.x version

The changes between 2.4.1 (May 2021) and 2.4.2 (April 2022) make sure that the 2.4.x line can still be built, as some of its dependencies have changed locations.  New features and bug fixes are only going into the currently maintained version, with the exception that some changes will be made to the last version of a previous major release of Kong Gateway once a new version arrives on the horizon.  We're currently busy with getting 3.0.0 out of the door, so some things are still going into 2.8.x, but the fix you're looking for has not made it into that bucket so far.

Hope this helps,
Hans
